tests/test1079.py

```diff
@@ -244,11 +245,15 @@ def testFitsHeader(self):
             subImg.writeFits(outFile)
             hdr = afwImage.readMetadata(outFile)
             
-            self.assertTrue( hdr.exists("LTV1"), "LTV1 not saved to fits header")
-            self.assertTrue( hdr.exists("LTV2"), "LTV2 not saved to fits header")
-            self.assertEqual(hdr.get("LTV1"), -1*x0, "LTV1 has wrong value")
-            self.assertEqual(hdr.get("LTV2"), -1*y0, "LTV1 has wrong value")
-
+            def checkLtvHeader(hdr, name, value):
```

Comment:
I guess it's just a style preference.  I don't see our coding standard suggests one way or another (or does it?) So don't worry then!